types/vis/vis-tests.ts

```diff
@@ -132,14 +135,14 @@ const dataset = new vis.DataSet<TestData>();
 
 // retrieve all items having a property group with value 2
 const group2 = dataset.get({
-  filter: (item) => {
+  filter: (item: any) => {
     return (item.group === 2);
   }
 });
 
 // retrieve all items having a property balance with a value above zero
 const positiveBalance = dataset.get({
-  filter: (item) => {
+  filter: (item: any) => {
```

Comment:
do not use any

Reply:
This in fact is not related with this pull request, but hooks (which have rules newer than latest vis change) are failing because implicit type warning. This is just to make hooks happy, I really dont think this should be a blocker for this PR because in fact i dont know what should be there.